MathlibTest/CommandStart.lean

```diff
@@ -100,8 +100,8 @@ example  :  True :=trivial
 /--
 warning: missing space
 
-Current syntax:  'le (a: Nat'
-Expected syntax: 'le (a : Na'
+Current syntax:  '(a: Nat)'
+Expected syntax: '(a : Nat'
```

Comment:
Can you also make this include the closing bracket?

Reply:
I fixed this, thanks!